tests/contrib/flask/__init__.py

```diff
@@ -7,13 +8,33 @@
 from tests.utils import TracerTestCase
 
 
+class DDFlaskTestClient(FlaskClient):
+    def __init__(self, *args, **kwargs):
+        super(DDFlaskTestClient, self).__init__(*args, **kwargs)
+
+    def open(self, *args, **kwargs):
```

Comment:
:+1: i'm good with this just because we have real integration tests with uwsgi and those demonstrate that this issue is scoped to the flask test client